docs/integrations/mlflow.rst

```diff
@@ -3,54 +3,91 @@
 Neptune-MLflow Integration
 ==========================
 
-|neptune-mlflow| is an open source project curated by Neptune team that enables |mlflow| experiment runs to be hosted in Neptune.
+|Run on Colab|
+
+What will you get with this integration?
+----------------------------------------
+
+|mlflow-tour|
+
+|neptune-mlflow| is an open source project curated by Neptune team that enables |mlflow| experiment runs to be hosted on Neptune.
 
 The integration lets you enjoy the best of both worlds: the tracking and reproducibility of MLflow with the organization and collaboration of Neptune.
```

Comment:
I think we should add a list of objects that will be converted from MLflow run to Neptune experiment.
those are:

- experiment id
- experiment name
- run id
- run name
- metrics
- parameters
- artifacts
- tags